core/src/transaction_meta.rs

```diff
@@ -22,36 +22,47 @@ impl From<BitVecSerde> for BitVec {
 pub struct TransactionMeta {
     block_number: u64,
     epoch_number: u64,
-    /// first bit indicate if transaction is a cellbase transaction
-    /// next bits indicate if transaction has dead cells
+    cellbase: bool,
+    /// each bits indicate if transaction has dead cells
     #[serde(with = "BitVecSerde")]
     dead_cell: BitVec,
```

Comment:
It names as `dead_cell` but actually means "output cells"? 
:(

Reply:
`rename` can be another PR (or commit). I just keep the old name. :)